src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -6,13 +6,13 @@
 5
 
 >> Value(ParseJson("true"))
```

Comment:
>Value [](http://example.com/codeflow?start=3&length=5)

can we add a comment here to explain why it's an error, eg:

// Value() doesn't parse and must be a json number.